components/Match/analysis.js

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable */
+import util from 'util';
+import constants from 'dotaconstants';
+
+function generatePlayerAnalysis(match, pm)
+{
+  // define condition check for each advice point
+  const advice = {};
+  const checks = {
+    // EFF@10
```

Comment:
could you comment or change the name (at least internal name) to something besides eff? it's not readily apparent what it is until you read the name attribute of the object returned.